client/devdocs/controller.js

```diff
@@ -1,43 +1,45 @@
 /**
  * External dependencies
  */
-var ReactDom = require( 'react-dom' ),
-	React = require( 'react' ),
-	qs = require( 'qs' ),
-	debounce = require( 'lodash/debounce' ),
-	page = require( 'page' ),
-	ReduxProvider = require( 'react-redux' ).Provider,
-	setSection = require( 'state/ui/actions' ).setSection,
-	EmptyContent = require( 'components/empty-content' );
+import ReactDom from 'react-dom';
+import React from 'react';
+import qs from 'qs';
+import debounce from 'lodash/debounce';
+import page from 'page';
+import { Provider } from 'react-redux';
```

Comment:
I essentially, wanted to keep the naming consistent i.e., name of imported module should tie out to name of exported module (as is the convention being followed across the codebase). This will also avoid any funky scenarios where (say) a new ReduxProvider module gets exported in 'react-redux'.

I recently learnt of the `import as` construct, so will use that as you have suggested as well, since I agree that the `ReduxProvider` name is much better.